### DISCUSSION TITLE
your main.yml

### DISCUSSION THREAD

**Discussion by zuckung** [Hecter94/EndlessSky-PluginArchive#67] [Ideas]
First, this is a great repository, found many interesting plugins here, thanks you for that. 

I have checked your main.yml and used it for my own repository and found a non-optimal function in it. When changing a plugin, the zip which is made contains your additional directories "Working/All Plugins/".

This code fixes that
```python
os.chdir('Working/All Plugins/')

subprocess.run(["zip", "-r", "../../" + p + ".zip", p], 

  stdout=subprocess.DEVNULL)

os.chdir('../../') 
```

**Comment by Hecter94** [comment]
Thanks for taking a look; if you PR the change, I'll look at it and see about committing it.

**Comment by zuckung** [comment]
I read the discord chat about your work on that action for overwriting existing artifacts. Found something? I'm pretty new to github and tried for hours to get that release action doing that. changing work paths in a python script was my greatest success :/

**Comment by zuckung** [comment]
Searched and searched, tried workarounds, different actions from the marketplace, console commands etc... nothing worked, till i found out that this behavior is a bug/feature from 2017 and how it works. Anyway, solved. Will PR the yml file soon.
